package_scripts/subset-fonts.ts

```diff
@@ -0,0 +1,147 @@
+/**
+ * The fundamentals for this utility are derived from
+ * https://stackoverflow.com/questions/64614572/creating-a-material-icons-subset
+ */
+
+import * as fs from "fs"
+import * as path from "path"
+import { promisify } from "util"
+import { exec } from "child_process"
+import { program } from "commander"
+
+const readdirAsync = promisify(fs.readdir)
+const statAsync = promisify(fs.stat)
+const execAsync = promisify(exec)
+const readFileAsync = promisify(fs.readFile)
```

Comment:
If I remember correctly, you can import from `fs/promises` for promisified versions.

Actually, importing from `node:fs/promises` might be mildly preferable. https://stateful.com/blog/node-18-prefix-only-modules